polkadot-protocol/basics/accounts.md

```diff
@@ -1,4 +1,328 @@
+    - By default, the Polkadot-SDK runtime configures the `AccountInfo` structure, as defined in the [Balances pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_balances/struct.AccountData.html){target=\_blank}
+
+## Account Balance Types
+
+In the Polkadot ecosystem, account balances are categorized into different types based on how the funds are utilized and their availability. These balance types determine the actions that can be performed, such as transferring tokens, paying transaction fees, or participating in governance activities. Understanding these balance types helps developers manage user accounts and implement balance-dependent logic.
+
+!!! tip "A more efficient distribution of account balance types is in development"
+    Soon, pallets in the Polkadot SDK will implement the [`fungible` trait](https://paritytech.github.io/polkadot-sdk/master/frame_support/traits/tokens/fungible/index.html){target=\_blank} (see the [tracking issue](https://github.com/paritytech/polkadot-sdk/issues/226){target=\_blank} for more details). This update will enable more efficient use of account balances, allowing the free balance to be utilized for on-chain activities such as setting proxies and managing identities.
+
+### Balance Types
+
+The five main balance types are:
+
+- **Free balance** - this balance represents the total tokens available to the account for any on-chain activity, including staking, governance, and voting. However, it may not be fully spendable or transferrable if portions of it are locked or reserved
+- **Locked balance** - locked balances are portions of the free balance that cannot be spent or transferred because they are tied up in specific activities like [staking](https://wiki.polkadot.network/docs/learn-staking#nominating-validators){target=\_blank}, [vesting](TODO: path to Vested Transfers subsection in Types of Txs), or participating in [governance](https://wiki.polkadot.network/docs/learn-polkadot-opengov#voting-on-a-referendum){target=\_blank}. While the tokens remain part of the free balance, they are non-transferable for the duration of the lock
+- **Reserved balance** - reserved balances are funds locked by specific system actions, such as setting up an [identity](https://wiki.polkadot.network/docs/learn-identity){target=\_blank}, creating [proxies](https://wiki.polkadot.network/docs/learn-proxies){target=\_blank}, or submitting [deposits for governance proposals](https://wiki.polkadot.network/docs/learn-guides-polkadot-opengov#claiming-opengov-deposits){target=\_blank}. These tokens are not part of the free balance and cannot be spent unless they are unreserved.
```

Comment:
```suggestion
- **Free balance** - represents the total tokens available to the account for any on-chain activity, including staking, governance, and voting. However, it may not be fully spendable or transferrable if portions of it are locked or reserved
- **Locked balance** - portions of the free balance that cannot be spent or transferred because they are tied up in specific activities like [staking](https://wiki.polkadot.network/docs/learn-staking#nominating-validators){target=\_blank}, [vesting](TODO: path to Vested Transfers subsection in Types of Txs), or participating in [governance](https://wiki.polkadot.network/docs/learn-polkadot-opengov#voting-on-a-referendum){target=\_blank}. While the tokens remain part of the free balance, they are non-transferable for the duration of the lock
- **Reserved balance** - funds locked by specific system actions, such as setting up an [identity](https://wiki.polkadot.network/docs/learn-identity){target=\_blank}, creating [proxies](https://wiki.polkadot.network/docs/learn-proxies){target=\_blank}, or submitting [deposits for governance proposals](https://wiki.polkadot.network/docs/learn-guides-polkadot-opengov#claiming-opengov-deposits){target=\_blank}. These tokens are not part of the free balance and cannot be spent unless they are unreserved
```